src/applications/mhv/medications/containers/LandingPage.jsx

```diff
@@ -1,23 +1,126 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { getPrescriptionsList } from '../actions/prescriptions';
 import MedicationsList from '../components/MedicationsList/MedicationsList';
 import MedicationsListSort from '../components/MedicationsList/MedicationsListSort';
+import { dateFormat, generateMedicationsPDF } from '../util/helpers';
 import PrintHeader from './PrintHeader';
 
 const LandingPage = () => {
+  const currentDate = new Date();
   const prescriptions = useSelector(
     state => state.rx.prescriptions.prescriptionsList,
   );
-  const dispatch = useDispatch();
+  const userName = useSelector(state => state.user.profile.userFullName);
+  const dob = useSelector(state => state.user.profile.dob);
 
+  const dispatch = useDispatch();
+  const [rxList, setRxList] = useState([]);
   useEffect(
     () => {
       dispatch(getPrescriptionsList());
```

Comment:
Is there any error handling here? If the api call fails do we bubble something up to the user or does it continue to show the loading indicator?

Reply:
there is not a flow in place, it would just show the loading spinner. I created a ticket for UCD to give us a flow for this scenario. https://jira.devops.va.gov/browse/MHV-47094

Reply:
great, thanks!